### ISSUE TITLE
auth failure

### ISSUE THREAD

**Issue by rados4y** [petergardfjall/garminexport#31]
I receive "failed with exception: auth failure: unable to extract auth ticket URL. did you provide a correct username/password?" with latest version.

**Comment by AngelaKru** [comment]
'reponse_url' is definitely missing from the sign in response in Garmin Connect API - worked fine for me last week, today it' missing. Did Garmin make changes in the last few days?

**Comment by rados4y** [comment]
Seems so, it was working fine so far.

**Comment by m-soltys** [comment]
It was working fine yesterday and stopped to work today. 

The 'Referer' header seems to be required in the authentication request. This line of code is fixing the problem.

```python
self.session.headers.update({'Referer': 'https://sso.garmin.com/sso/signin?service=https://connect.garmin.com/modern/'})
```
You has to add it to garminclient.py line 114.

**Comment by petergardfjall** [comment]
Appears like Garmin switched their signin endpoint. The latest commit should fix it: beeb052b
